azure/README.md

```diff
@@ -17,37 +17,18 @@ These images later can be used together with deployment scripts above in order t
 To copy CyberArk Privileged Access Security solution snapshot and create the images in your Azure subscription, use the import-pas-images.ps1 PowerShell script. Make sure you have all the CyberArk components AccessSAS URLs before executing the script.
 
 ## Usage
-As a first step customer must receive components AccessSAS.
+As a prerequisite to deploy PAM components using the supplied ARM templates, customer must have the compatible images ready in a designated Resource Group in the destination subscription.
+
+This can be achieved by downloading the import-pas-images.ps1 from CyberArk's Marketplace at:
+https://cyberark.my.site.com/mplace/s/#software
+You should navigate "Privileged Access Manager Self-Hosted" matching your desired release version.
```

Comment:
You should navigate "Privileged Access Manager Self-Hosted" => You should navigate to "Privileged Access Manager Self-Hosted"